### ISSUE TITLE
paddleocr 训练时使用多卡多进程训练时CPU使用率不稳定，进程数不稳定

### ISSUE THREAD

**Issue by QQQTAO** [PaddlePaddle/PaddleOCR#6975]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：2.3
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：2.5
- 运行指令/Command Code：[2022/07/21 11:45:07] ppocr INFO: Architecture : 
[2022/07/21 11:45:07] ppocr INFO:     Backbone : 
[2022/07/21 11:45:07] ppocr INFO:         layers : 34
[2022/07/21 11:45:07] ppocr INFO:         name : ResNet
[2022/07/21 11:45:07] ppocr INFO:     Head : 
[2022/07/21 11:45:07] ppocr INFO:         fc_decay : 4e-05
[2022/07/21 11:45:07] ppocr INFO:         name : CTCHead
[2022/07/21 11:45:07] ppocr INFO:     Neck : 
[2022/07/21 11:45:07] ppocr INFO:         encoder_type : rnn
[2022/07/21 11:45:07] ppocr INFO:         hidden_size : 256
[2022/07/21 11:45:07] ppocr INFO:         name : SequenceEncoder
[2022/07/21 11:45:07] ppocr INFO:     Transform : None
[2022/07/21 11:45:07] ppocr INFO:     algorithm : CRNN
[2022/07/21 11:45:07] ppocr INFO:     model_type : rec
[2022/07/21 11:45:07] ppocr INFO: Eval : 
[2022/07/21 11:45:07] ppocr INFO:     dataset : 
[2022/07/21 11:45:07] ppocr INFO:         data_dir : 
[2022/07/21 11:45:07] ppocr INFO:         label_file_list : ['/annotated_data/new_label_test.txt']
[2022/07/21 11:45:07] ppocr INFO:         name : SimpleDataSet
[2022/07/21 11:45:07] ppocr INFO:         transforms : 
[2022/07/21 11:45:07] ppocr INFO:             DecodeImage : 
[2022/07/21 11:45:07] ppocr INFO:                 channel_first : False
[2022/07/21 11:45:07] ppocr INFO:                 img_mode : BGR
[2022/07/21 11:45:07] ppocr INFO:             CTCLabelEncode : None
[2022/07/21 11:45:07] ppocr INFO:             RecResizeImg : 
[2022/07/21 11:45:07] ppocr INFO:                 image_shape : [3, 32, 320]
[2022/07/21 11:45:07] ppocr INFO:             KeepKeys : 
[2022/07/21 11:45:07] ppocr INFO:                 keep_keys : ['image', 'label', 'length']
[2022/07/21 11:45:07] ppocr INFO:     loader : 
[2022/07/21 11:45:07] ppocr INFO:         batch_size_per_card : 256
[2022/07/21 11:45:07] ppocr INFO:         drop_last : False
[2022/07/21 11:45:07] ppocr INFO:         num_workers : 4
[2022/07/21 11:45:07] ppocr INFO:         shuffle : False
[2022/07/21 11:45:07] ppocr INFO: Global : 
[2022/07/21 11:45:07] ppocr INFO:     cal_metric_during_train : True
[2022/07/21 11:45:07] ppocr INFO:     character_dict_path : ppocr/utils/ppocr_keys_v1.txt
[2022/07/21 11:45:07] ppocr INFO:     checkpoints : None
[2022/07/21 11:45:07] ppocr INFO:     distributed : True
[2022/07/21 11:45:07] ppocr INFO:     epoch_num : 200
[2022/07/21 11:45:07] ppocr INFO:     eval_batch_step : [0, 2000]
[2022/07/21 11:45:07] ppocr INFO:     infer_img : doc/imgs_words/ch/word_1.jpg
[2022/07/21 11:45:07] ppocr INFO:     infer_mode : False
[2022/07/21 11:45:07] ppocr INFO:     log_smooth_window : 20
[2022/07/21 11:45:07] ppocr INFO:     max_text_length : 120
[2022/07/21 11:45:07] ppocr INFO:     pretrained_model : output/ch_ppocr_v2.0_7.20/best_accuracy.pdparams
[2022/07/21 11:45:07] ppocr INFO:     print_batch_step : 10
[2022/07/21 11:45:07] ppocr INFO:     save_epoch_step : 1
[2022/07/21 11:45:07] ppocr INFO:     save_inference_dir : None
[2022/07/21 11:45:07] ppocr INFO:     save_model_dir : ./output/ch_ppocr_v2.0_7.21
[2022/07/21 11:45:07] ppocr INFO:     save_res_path : ./output/rec/predicts_chinese_common_v2.0.txt
[2022/07/21 11:45:07] ppocr INFO:     use_gpu : True
[2022/07/21 11:45:07] ppocr INFO:     use_space_char : True
[2022/07/21 11:45:07] ppocr INFO:     use_visualdl : False
[2022/07/21 11:45:07] ppocr INFO: Loss : 
[2022/07/21 11:45:07] ppocr INFO:     name : CTCLoss
[2022/07/21 11:45:07] ppocr INFO: Metric : 
[2022/07/21 11:45:07] ppocr INFO:     main_indicator : acc
[2022/07/21 11:45:07] ppocr INFO:     name : RecMetric
[2022/07/21 11:45:07] ppocr INFO: Optimizer : 
[2022/07/21 11:45:07] ppocr INFO:     beta1 : 0.9
[2022/07/21 11:45:07] ppocr INFO:     beta2 : 0.999
[2022/07/21 11:45:07] ppocr INFO:     lr : 
[2022/07/21 11:45:07] ppocr INFO:         learning_rate : 0.0002
[2022/07/21 11:45:07] ppocr INFO:         name : Cosine
[2022/07/21 11:45:07] ppocr INFO:         warmup_epoch : 5
[2022/07/21 11:45:07] ppocr INFO:     name : Adam
[2022/07/21 11:45:07] ppocr INFO:     regularizer : 
[2022/07/21 11:45:07] ppocr INFO:         factor : 4e-05
[2022/07/21 11:45:07] ppocr INFO:         name : L2
[2022/07/21 11:45:07] ppocr INFO: PostProcess : 
[2022/07/21 11:45:07] ppocr INFO:     name : CTCLabelDecode
[2022/07/21 11:45:07] ppocr INFO: Train : 
[2022/07/21 11:45:07] ppocr INFO:     dataset : 
[2022/07/21 11:45:07] ppocr INFO:         data_dir : /data/wangkuan/data/rec_dataset
[2022/07/21 11:45:07] ppocr INFO:         ext_op_transform_idx : 1
[2022/07/21 11:45:07] ppocr INFO:         label_file_list : ['/annotated_data/new_label_train.txt']
[2022/07/21 11:45:07] ppocr INFO:         name : SimpleDataSet
[2022/07/21 11:45:07] ppocr INFO:         transforms : 
[2022/07/21 11:45:07] ppocr INFO:             DecodeImage : 
[2022/07/21 11:45:07] ppocr INFO:                 channel_first : False
[2022/07/21 11:45:07] ppocr INFO:                 img_mode : BGR
[2022/07/21 11:45:07] ppocr INFO:             RecConAug : 
[2022/07/21 11:45:07] ppocr INFO:                 ext_data_num : 2
[2022/07/21 11:45:07] ppocr INFO:                 image_shape : [32, 320, 3]
[2022/07/21 11:45:07] ppocr INFO:                 prob : 0.4
[2022/07/21 11:45:07] ppocr INFO:             RecAug : None
[2022/07/21 11:45:07] ppocr INFO:             CTCLabelEncode : None
[2022/07/21 11:45:07] ppocr INFO:             RecResizeImg : 
[2022/07/21 11:45:07] ppocr INFO:                 image_shape : [3, 32, 320]
[2022/07/21 11:45:07] ppocr INFO:             KeepKeys : 
[2022/07/21 11:45:07] ppocr INFO:                 keep_keys : ['image', 'label', 'length']
[2022/07/21 11:45:07] ppocr INFO:     loader : 
[2022/07/21 11:45:07] ppocr INFO:         batch_size_per_card : 256
[2022/07/21 11:45:07] ppocr INFO:         drop_last : True
[2022/07/21 11:45:07] ppocr INFO:         num_workers : 4
[2022/07/21 11:45:07] ppocr INFO:         shuffle : False
[2022/07/21 11:45:07] ppocr INFO: profiler_options : None
- 完整报错/Complete Error Message：
- 使用5张卡和进程数为4进行训练，训练进程数在5到20之间波动，每次训练迭代时间不稳定，是否可以通过设置共享内存大小或者其他设置解决这种情况，训练时如何设置共享内存大小

**Comment by QQQTAO** [comment]
![QQ截图20220722173901](https://user-images.githubusercontent.com/48081128/180414748-7c639e82-08ed-4476-a7c8-6d658a593c65.jpg)

**Comment by LDOUBLEV** [comment]
训练的时候无法指定共享内存大小
